src/Libraries/Module/Templates/Demo/Web/Middlewares/Password.tpl

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * Quantum PHP Framework
+ *
+ * An open source software development framework for PHP
+ *
+ * @package Quantum
+ * @author Arman Ag. <arman.ag@softberg.org>
+ * @copyright Copyright (c) 2018 Softberg LLC (https://softberg.org)
+ * @link http://quantum.softberg.org/
+ * @since 2.9.5
+ */
+
+namespace {{MODULE_NAMESPACE}}\Middlewares;
+
+use Quantum\Libraries\Validation\Validator;
+use Quantum\Libraries\Validation\Rule;
+use Quantum\Libraries\Hasher\Hasher;
+use Quantum\Middleware\QtMiddleware;
+use Quantum\Factory\ModelFactory;
+use Quantum\Http\Response;
+use Quantum\Http\Request;
+use Shared\Models\User;
+use Closure;
+
+/**
+ * Class Password
+ * @package Modules\Web
+ */
+class Password extends QtMiddleware
+{
+    /**
+     * @var Validator
+     */
+    private $validator;
+
+    /**
+     * Class constructor
+     * @throws \Exception
+     * @param Request $request
+     */
+    public function __construct(Request $request)
+    {
+        $this->validator = new Validator();
+        $hasher = new Hasher();
+        $hasher->setAlgorithm(PASSWORD_BCRYPT);
+        $user = ModelFactory::get(User::class)->findOneBy('uuid', $request->get('uuid', null));
```

Comment:
User should not be relayed on UUID from request, the logged in user UUID must be used

Reply:
not fixed